code/stylesheets/spec.pcss

```diff
@@ -82,7 +82,11 @@
     }
 
     ul {
-      @apply list-disc list-inside my-4
+      @apply list-disc list-outside my-4
     }
   }
 }
+
+.spec main ul {
+  margin-left: 1em;
+}
```

Comment:
The 1.1 spec has a multi-paragraph now (in "Preliminaries") and renders to `<li><p>...</p></li>`. This looks horrible with `list-style-position: inside`, so change that to the default (outside). Also add a margin which gets reset otherwise.